profiler/profiler_test.go

```diff
@@ -26,19 +26,19 @@ func (s TestSuite) TestProfilerServiceOperations(c *C) {
 
 	testCases := map[string]testCase{
 		"ProfilerOP(...): show": {
-			op:         "show",
+			op:         OpSHOW,
 			portNumber: 0,
```

Comment:
I'm not sure if we want to consider reusing the Op_name from upstream. WDYT?
```
profilerpb.Op_name[int32(profilerpb.Op_SHOW)]
```

Reply:
Because I would like to test the caller as below
```
_, err := client.ProfilerOP(testCase.op, testCase.portNumber)
```

This allows the user to use a human-readable string to operate.
So, I keep the same usage as a normal use case.

Reply:
The return value `profilerpb.Op_name[int32(profilerpb.Op_SHOW)]` is same as `OpSHOW`, without the necessity of introducing additional constants.

But maybe reusing Op_name is a bit too complex.

Reply:
yes, a little bit complex. I prefer to use internal variables for it.

Updated with the latest changes; please help to recheck it. Thanks!